cnv_somatic_panel_workflow.b37.inputs.json

```diff
@@ -0,0 +1,48 @@
+{
+  "##_COMMENT1": "Inputs",
+  "CNVSomaticPanelWorkflow.normal_bams": ["gs://gatk-test-data/cnv/somatic/SM-74NEG.bam", "gs://gatk-test-data/cnv/somatic/SM-74NF5.bam"],
```

Comment:
Note that you're building a panel of normals from 1 normal sample (SM-74NEG) and 1 tumor sample (SM-74NF5).  Moreover, the normal sample has a large CNV event in chr2, which is not the sort of sample we'd want to include in the PoN in a real analysis.  Finally, you are including the same normal in the pair workflow, which may cause unusual behavior in a PoN this small---basically, we are using the PoN to learn what sequencing noise looks like, and when we have only two samples both will influence the result heavily.  If we then use this PoN to denoise one of those samples, we will basically think it looks like sequencing noise!

So let's step back a bit---can I ask what the goal of these JSONs is?  Is it to demonstrate what results might reasonably look like on a small set of real publicly available data?  Are we trying to demonstrate "best practices" in any sense?  Or is it just to test that the workflows will run on real data (rather than downsampled snippets of test data)?

If it's anything but the latter, we should probably choose normal samples that are more representative of a realistic analysis.  These samples should be sequenced in a similar manner as those used for the pair.

Reply:
I think when the inputs were originally chosen, it was 100% only to check the WDL logic was functional. As we've not had regularly had scientific reviewers for those workflow source, we've not been careful about selecting inputs that represent a scientific best practices and left that burden to the user bringing their data/reading the docs about what the tools actually do. Since the review process now requires a scientific review, hopefully we can begin correcting any such issues.

Reply:
OK.  Just to be clear, if we're just testing that the workflows will run on real-sized data, then this is fine!  But if these are supposed to be pedagogical examples of how to run the workflows, then we can pick more appropriate samples so that users aren't confused.

Reply:
I think it would be a great improvement if these workflows can transition into pedagogical examples of how to run the workflows -- this way the data coming out of these workflows (even on a test run) is meaningful. You may want to check out gs://gatk-test-data/cnv/somatic  to see the data available on the cloud, and gage what needs to be added there to fulfill input requirements for the CNV workflows.

Reply:
OK, I also think that'd be great!  For the PoN, let's try using two publicly available WES non-cell line normals (there may be some already in the test repo, e.g. NA12878 replicates?)  If these give reasonable results on the HCC1143 pair (we can just check the resulting plots by eye), then we'll have a nice self contained example.  Of course, tutorials or any other supporting materials should emphasize that more samples should be used in a PoN.

Reply:
Hey Sam,
Prepping the dataset is taking longer than expected. For the time being I'll do a merge to keep the wdls up to date. But will work on getting proper datasets ready.